code/__DEFINES/supermatter.dm

```diff
@@ -2,7 +2,7 @@
 #define OXYGEN_HEAT_PENALTY 1
 #define PLUOXIUM_HEAT_PENALTY -0.5 //Better then co2, worse then n2
 #define TRITIUM_HEAT_PENALTY 10
-#define CO2_HEAT_PENALTY 6
+#define CO2_HEAT_PENALTY 2
```

Comment:
Thinking about it overnight, this might lead to some interesting gameplay options. The only thing is the loss of co2 power transmittance increase, you could have it multiplied by a quarter so that co2 still retains some interesting quirks that make it much better than an average N2 SM